blog/2024-11-20-integration-quality-scale.md

```diff
@@ -0,0 +1,29 @@
+---
+author: Joost Lekkerkerker
+authorURL: https://github.com/joostlek
+authorImageURL: https://avatars.githubusercontent.com/u/7083755?v=4
+title: "Integration quality scale"
+---
+
+Since the original quality scale has been introduced in 2020, Home Assistant changed quite a lot.
+Both in how integrations are developed and what features they can provide.
+The quality scale, however, has not been updated to reflect these changes.
```

Comment:
```suggestion
Since the original quality scale was introduced in 2020, Home Assistant changed quite a lot.
Both in how integrations are developed and what features they can provide.
The quality scale, however, has not been updated to reflect these changes.
```